spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -186,7 +185,7 @@ public void testCustomPartitionCountOverridesDefaultIfLarger() throws Exception
 		// Let the consumer actually bind to the producer before sending a msg
 		binderBindUnbindLatency();
 		moduleOutputChannel.send(message);
-		Message<?> inbound = moduleInputChannel.receive(2000);
+		Message<?> inbound = moduleInputChannel.receive((int)(2000 * timeoutMultiplier));
```

Comment:
The constant is not ... constant.
Even though in a given test it is often (arbitrarily) the same value (2000 in the snippet here), it could be different.

So really, should be `receiveWith(channel, baseAmount)`. Do we gain anything?